src/lang/modifyAst/addFillet.ts

```diff
@@ -253,6 +259,11 @@ async function updateAstAndFocus(
   const updatedAst = await kclManager.updateAst(modifiedAst, true, {
     focusPath: pathToFilletNode,
   })
+
+  const newCode = recast(updatedAst.newAst)
+  if (err(newCode)) return
+  await codeManager.updateCodeEditor(newCode)
```

Comment:
Is there a reason this doesn't write to disk?

Reply:
Looks like one I missed, really good eye.